app/javascript/crayons/MultiSelectAutocomplete/MultiSelectAutocomplete.jsx

```diff
@@ -66,6 +66,7 @@ const reducer = (state, action) => {
  * @param {number} props.maxSelections Optional maximum number of allowed selections
  * @param {Function} props.onSelectionsChanged Callback for when selections are added or removed
  * @param {Function} props.onFocus Callback for when the component receives focus
+ * @param {boolean} props.allowUserDefinedSelections Whether a user can create new selections other than the defined suggestions
```

Comment:
Maybe there is a better name for this prop - I couldn't think of one, but I'm very open to suggestions!